docs/docs_src/tutorials/whatsapp/main.py

```diff
@@ -21,9 +21,11 @@
     "temperature": 0.8,
 }
 
-openapi_url = "https://raw.githubusercontent.com/airtai/fastagency/refs/heads/main/examples/openapi/whatsapp_openapi.json"
+openapi_url = "https://dev.infobip.com/openapi/products/whatsapp.json"
+
 whatsapp_api = OpenAPI.create(
     openapi_url=openapi_url,
+    servers=[{"url": "https://api.infobip.com"}],
```

Comment:
```suggestion
    # this is an optional parameter, but specified here because servers are not specified in the OpenAPI specification
    servers=[{"url": "https://api.infobip.com"}],
```